roles/splunk_standalone/molecule/snowflake/group_vars/all.yml

```diff
@@ -1,6 +1,10 @@
 ---
 splunk:
   role: splunk_standalone
+  appserver:
+    port: 8066
+  kvstore:
+    port: 18191
```

Comment:
Is this intentionally 18191 instead of 8191?

Reply:
Yeah, the `snowflake` scenario tests the ability to change the kvstore port and makes sure that the change is reflected: https://github.com/splunk/splunk-ansible/pull/470/files#diff-f6dc38764b2c3623237a3a6e7c0e5734R106